server/bleep/src/indexes/file.rs

```diff
@@ -408,42 +423,40 @@ impl File {
         };
         let entry_pathbuf = repo_disk_path.join(&relative_path);
 
-        let content_hash = {
+        let semantic_hash = {
```

Comment:
Do we need this `semantic_hash` if we select by `repo_name` and `relative_path` in Qdrant?

Reply:
As noted above, the combination of `repo_name` and `relative_path` is insufficient to uniquely identify resources when dealing with branches.